service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToSubmitTest.java

```diff
@@ -47,7 +51,7 @@ void shouldReturnUnsuccessfulResponseWithMalformedData() {
     }
 
     @Test
-    void shouldReturnSuccessfulResponseWithValidCaseData() {
+    void shouldAppendDocumentAndToggleCtscOffWhenProvidedValidCaseDataAndCtscIsToggledOff() {
```

Comment:
this really confuses me. Lots going on 😕

Reply:
Difficult to rename as there is a lot going on in this test. I've opted to not create an additional int test and went for `shouldSetCtscPropertyToYesWhenCtscLaunchDarklyVariableIsEnabled`. Thoughts?